src/awsexplorer/regionNode.ts

```diff
@@ -71,7 +72,16 @@ export class RegionNode extends AWSTreeNodeBase {
         }
     }
 
+    private tryClearChildren(): void {
+        this.childNodes.forEach(cn => {
+            if ('clearChildren' in cn) {
+                ;(cn as AWSTreeNodeBase & LoadMoreNode).clearChildren()
+            }
+        })
+    }
+
     public async getChildren(): Promise<AWSTreeNodeBase[]> {
```

Comment:
Is there a scenario where I want to refresh but not clear the cache?

Reply:
Yes, when you want to load a new page.  In that case, the node is passed to the refresh command, but because it didn't stem from the Region node, it will not clear the cache.